gui/src/app/pages/HomePage/BrowserProjectsInterface.ts

```diff
@@ -0,0 +1,108 @@
+export class BrowserProjectsInterface {
+  constructor(
+    private dbName: string = "stan-playground",
+    private storeName: string = "projects",
+  ) {}
+  async loadProject(title: string) {
+    const db = await this.openDatabase();
+    const transaction = db.transaction(this.storeName, "readonly");
+    const objectStore = transaction.objectStore(this.storeName);
```

Comment:
We  need to repeat these steps for basically any connection to the db (loading, saving, listing, or deleting) so might be nice to pull out into a function that just returns the `objectStore`.

Reply:
done